test/data/wfs.py

```diff
@@ -0,0 +1,386 @@
+def get_wfs_insert_points(username, layername):
```

Comment:
now, when we use more WFS versions, what about naming methods in this file
`get_wfs<version>_...`, e.g. `get_wfs10_...`,  get_wfs20_...`.